iphone/Maps/Maps.xcodeproj/project.pbxproj

```diff
@@ -5179,6 +5201,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 0;
 				DEVELOPMENT_TEAM = 9Z6432XD7L;
+				ENABLE_USER_SCRIPT_SANDBOXING = NO;
```

Comment:
Can it be set in common.xcconfig instead of copying the same value in each configuration? Then has the benefit of reusing the same value when a new config is added later.

Reply:
Yes.